### PR TITLE
Jl server tests

### PR DESCRIPTION
**Pull request by dividedsky**
Adds tests for teachersRoutes and studentsRoutes.

Requesting Chaya's review bc I had to slightly alter a few db helpers.

### CODE REVIEW

File: server/data/helpers/classesHelper.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ module.exports = {
         'teachers.lastname as t_last',
         'teachers.email as t_email'
       )
-      .join('classes', 'teachers.id', 'classes.teacher_id')
-      .where('classes.id', id);
+      .join('teachers_classes', 'teachers.id', 'teachers_classes.teacher_id')
+      .where('teachers_classes.class_id', id);
```
Comment (cotikor) [inline server/data/helpers/classesHelper.js]: Did you add the many to many table for teachers_classes?  I wasn't sure if we were going to let one class have multiple teachers or not. I think it makes sense if classes are actually just sections kind of like Lambda sections can have multiple teachers.

Comment (dividedsky) [reply server/data/helpers/classesHelper.js]: Oh, my bad! It looks like I was still using my dev db, and I guess I forgot that we had removed teachers_classes in the new schema.

Kelly, can we hold off on merging this one? I will alter the tests so they work with the new schema.

File: server/data/helpers/classesHelper.js [hunk 2]
```diff
@@ -18,8 +18,8 @@ module.exports = {
         'teachers.lastname as t_last',
         'teachers.email as t_email'
       )
-      .join('teachers_classes', 'teachers.id', 'teachers_classes.teacher_id')
-      .where('teachers_classes.class_id', id);
+      .join('classes', 'teachers.id', 'classes.teacher_id')
+      .where('classes.id', id);
```
Comment (cotikor) [inline server/data/helpers/classesHelper.js]: Looks good!

**Comment by kelfro** [comment]
Great job on the testing!

**Pull request merged**